andela/titan-dev-ah-frontend
# #163518640 Set Up SassProcessor

#### What does this PR do?
- Setup sass processor
#### Description of Task to be completed?
- Include sass, css, and style loader
- Extract css into its own file for production
- File loader such as image, icons, and svg
#### How should this be manually tested?
- Fork the repo and check out the branch `ch-setup-sass-processor-163518640`
- In the root folder run `yarn` to install all dependencies
- Run `yarn start` to start the project
- The browser should be opened. Click the button label `Fetch more articles`. 
- You should see that scss are being load, and file such as image

#### What are the relevant pivotal tracker stories?
#163518640
